benchexec/tools/aise.py

```diff
@@ -0,0 +1,89 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
+# https://github.com/sosy-lab/benchexec
+#
+# SPDX-FileCopyrightText: 2007-2020 Dirk Beyer <https://www.sosy-lab.org>
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import benchexec.result as result
+import benchexec.tools.template
+import benchexec.model
+
+
+class Tool(benchexec.tools.template.BaseTool2):
+    """
+    Tool info for AISE.
```

Comment:
Please add a URL for your tool here. AISE seems to be a quite common abbreviation and it seems difficult to find your tool via Google.

Reply:
Ok, I will add a URL.